internal/util/projutil/project_util.go

```diff
@@ -82,7 +82,7 @@ func (e ErrInvalidDepManager) Error() string {
 	return fmt.Sprintf(`"%s" is not a valid dep manager; dep manager must be one of ["%v", "%v"]`, string(e), DepManagerDep, DepManagerGoMod)
 }
 
-var ErrNoDepManager = fmt.Errorf(`no valid dependency manager file found; dep manager must be one of ["%v", "%v"]`, DepManagerDep, DepManagerGoMod)
+var ErrNoDepManager = fmt.Errorf(`no valid dependency manager file was found; dep manager must be one of ["%v", "%v"]`, DepManagerDep, DepManagerGoMod)
```

Comment:
actually the *was* isn't needed. The error was more "computer speak". Similar to "file not found" errors. I would never say someone "file not found" or "item not found" but in an error message I would use those terms. So I would use the old value.

Reply:
OK. I got your point. Tks for the explanation 👍